src/style/index.scss

```diff
@@ -260,11 +260,15 @@ $landing-color-dark-bg: $color-hack-black-5;
 
```

Comment:
Remove all the commented styles

Reply:
this is all from @ambr0se6. I'll just go ahead and remove them